apollo-compiler/src/test/graphql/com/example/fragment_in_fragment/AllStarship.kt

```diff
@@ -233,8 +229,33 @@ class AllStarship : Query<AllStarship.Data, Operation.Variables> {
 
                         override fun marshaller(): ResponseFieldMarshaller
                       }
+
+                      companion object {
+                        fun Homeworld.planetFragment(): PlanetFragment? = this as? PlanetFragment
+
+                        fun Homeworld.pilotFragmentHomeworldPlanet():
+                            PilotFragment.Homeworld.Planet? = this as?
+                            PilotFragment.Homeworld.Planet
+
+                        fun Homeworld.starshipFragmentPilotConnectionEdgesNodePersonHomeworldPlanet():
```

Comment:
That sounds like a little bit too much 🙃 . Feels like this shouldn't be generated

Reply:
Yeah, I guess we need to skip generating accessors for nested named fragments. Only accessors for top level fragments.